tag/nitag.yml

```diff
@@ -0,0 +1,1816 @@
+          type: string
+          required: true
+      responses:
+        200:
+          description: Success
+          schema:
+            type: array
+            items:
+              description: Tag path with value
+              type: object
+              title: Tag Path With Value
+              properties:
+                path:
+                  type: string
+                  description: Tag path
+                current:
+                  $ref: '#/definitions/TagValue'
+        401:
+          $ref: '#/responses/Unauthorized'
+        404:
+          description: Selection with the given ID was not found
+        default:
+          $ref: '#/responses/Error'
+  /v2/selections/{id}/values/current/timestamp:
+    get:
+      tags:
+        - selections
+      summary: Get the timestamps of the current values of the tags in a selection
+      description: Gets the timestamps of the current values of the tags in a selection
+      operationId: GetSelectionValuesCurrentTimestamp
+      parameters:
+        - in: path
+          name: id
+          description: Selection ID
+          type: string
+          required: true
+      responses:
+        200:
+          description: Success
+          schema:
+            description: Array containing the paths of the tags in the selection, along with the timestamp of their current values
+            type: array
+            title: Tag Paths With Timestamps
+            items:
+              type: object
+              title: Tag Path and Timestamp
+              description: A tag path and the timestamp of its current value
+              properties:
+                timestamp:
+                  description: Timestamp associated with the current value
+                  type: string
+                path:
+                  type: string
+        401:
+          $ref: '#/responses/Unauthorized'
+        404:
+          description: Selection with the specified ID was not found
+        default:
+          $ref: '#/responses/Error'
+  /v2/selections/{id}/values/aggregates/min:
+    get:
+      tags:
+        - selections
+      summary: Get the "min" aggregate values for the tags in a selection
+      description: Returns the minimum aggregate values for the tags in a selection
+      operationId: GetSelectionMinValues
+      parameters:
+        - in: path
+          name: id
+          description: Selection ID
+          type: string
+          required: true
+      responses:
+        200:
+          description: Success
+          schema:
+            description: Array of the "min" values from the aggregates for each tag in the selection.
+            type: array
+            items:
+              type: object
+              title: Tag Paths and Count Values
+              properties:
+                path:
+                  type: string
+                min:
+                  description: Minimum value from the tag's aggregates
+                  type: number
```

Comment:
Throughout this API, tag aggregates changed from v1 to v2. In v2, aggregates are all strings (except count). In v1, they are doubles (except count is an integer). I don't know if SL Cloud has that change in their v2, but I would think so.

Applies to the other selection aggregate routes below.

Reply:
Cloud has not made that change. I filed them a CAR. We'll document them as strings.